owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/files/FileUtils.java

```diff
@@ -45,41 +45,15 @@ public static String getParentPath(String remotePath) {
      * : , " , | , ? , *
      *
      * @param fileName
-     * @param versionSupportsForbiddenChars
      * @return
      */
-    public static boolean isValidName(String fileName, boolean versionSupportsForbiddenChars) {
+    public static boolean isValidName(String fileName) {
         boolean result = true;
 
         Log_OC.d(TAG, "fileName =======" + fileName);
-        if ((versionSupportsForbiddenChars && fileName.contains(PATH_SEPARATOR)) ||
-                (!versionSupportsForbiddenChars && (fileName.contains(PATH_SEPARATOR) ||
-                        fileName.contains("\\") || fileName.contains("<") || fileName.contains(">") ||
-                        fileName.contains(":") || fileName.contains("\"") || fileName.contains("|") ||
-                        fileName.contains("?") || fileName.contains("*")))) {
-
-            result = false;
-        }
-        return result;
-    }
-
-    /**
-     * Validate the path to detect if contains any forbidden character: \ , < , > , : , " , | ,
-     * ? , *
-     *
-     * @param path
-     * @return
-     */
-    public static boolean isValidPath(String path, boolean versionSupportsForbidenChars) {
-        boolean result = true;
-
-        Log_OC.d(TAG, "path ....... " + path);
-        if (!versionSupportsForbidenChars &&
-                (path.contains("\\") || path.contains("<") || path.contains(">") ||
-                        path.contains(":") || path.contains("\"") || path.contains("|") ||
-                        path.contains("?") || path.contains("*"))) {
+        if (fileName.contains(PATH_SEPARATOR)) {
```

Comment:
Names with "\" or just an "*" are not allowed neither. Maybe we should check which characters are forbidden. What do you think, @davigonz ?

Reply:
Checking this...